core/services/chainlink/application.go

```diff
@@ -162,10 +161,12 @@ type ApplicationOpts struct {
 
 // Chains holds a ChainSet for each type of chain.
 type Chains struct {
-	EVM      evm.ChainSet
-	Cosmos   cosmos.ChainSet     // nil if disabled
-	Solana   solana.ChainSet     // nil if disabled
-	StarkNet starkchain.ChainSet // nil if disabled
+	EVM           evm.ChainSet
+	Cosmos        cosmos.ChainSet              // nil if disabled
+	Solana        solana.ChainSet              // Deprecated
+	SolanaService services.ServiceCtx          // nil if disabled
```

Comment:
comment about what differentiate Solana vs SolanaService and SolanaRelayer. are the latter 2 plugin specific?

maybe Solana(Plugin/LOOP)Service Solana(Plugin/LOOP)Relayer

Reply:
Simplified back to a single field.